<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>latlongDialogBase</class>
 <widget class="QDialog" name="latlongDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Latitude/Longitude calc</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Latitude: </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLatD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Degrees</string>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLatM">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Minutes</string>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLatS">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> Seconds</string>
   </property>
   <property name="maximum">
    <double>59.990000000000002</double>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLatH">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLatDD">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> DD</string>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLngM">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Minutes</string>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLngS">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> Seconds</string>
   </property>
   <property name="maximum">
    <double>59.990000000000002</double>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLngH">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>53</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLngDD">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string> DD</string>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLngD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Degrees</string>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Longitude: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
